.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         build_type: [Debug, Release]
         project_tags: [Stable, Unstable]
-        os: [ubuntu-latest, macOS-10.14]
+        os: [ubuntu-latest, macOS-10.14, windows-2019]
```

Comment:
I didn't find this key in the official documentation. Where did you find it?

Reply:
Apparently both windows-2019 and macOS-10.14 disappeared from the documentation, but they used to be there and they are still working fine. See https://github.com/actions/virtual-environments/issues/81 for a related issue for macOS .

Reply:
I see, thanks for the link!